debugger-launchjson.md

```diff
@@ -338,3 +338,16 @@ Example:
 ```json
     "targetArchitecture": "arm64"
 ```
+
+## Check for DevCert
+If unspecified, it will be enabled when `serverReadyAction` is set AND when `pipeTransport` is NOT set.
```

Comment:
>If [](http://example.com/codeflow?start=0&length=2)

I like the text you have, but I think we need start with an explanation of what development certificates are.

Maybe something similar to the text I suggested for OptionsSchema.json?

This option controls if, on launch, the the debugger should check if the computer has a self-signed HTTPS certificate used to develop web projects running on https endpoints.